packages/rollup/test/integration/far/a/BUILD.bazel

```diff
@@ -10,5 +10,5 @@ ts_project(
 js_library(
     name = "a",
     package_name = "@far/a",
-    deps = [":tsconfig"],
+    srcs = [":tsconfig"],
```

Comment:
This is an example change to show how a clear separation of sources vs. actual runtime deps shoukd be declared.. allowing for better determination of sources in `js_library`. This would also become relevant for supporting e.g. transitioned outputs in a `js_library`. Right now the fact that both `deps` and `srcs` is considered is really magical and unexpected IMO.